src/main/vcpf4/usbd_cdc_vcp.c

```diff
@@ -131,17 +131,19 @@ static uint16_t VCP_Ctrl(uint32_t Cmd, uint8_t* Buf, uint32_t Len)
       //Note - hw flow control on UART 1-3 and 6 only
       case SET_LINE_CODING:
          // If a callback is provided, tell the upper driver of changes in baud rate
-         if (plc && (Len == sizeof (*plc))) {
+         if (plc && (Len == sizeof (LINE_CODING))) {
              if (baudRateCb) {
```

Comment:
Same here

Reply:
Ditto

Reply:
Actually here `plc` is of type `LINE_CODING*` and it should remain `sizeof(*plc)`